scala-package/core/src/main/scala/org/apache/mxnet/FeedForward.scala

```diff
@@ -230,8 +230,11 @@ class FeedForward private(
       val padded = batch.pad
       val realSize = batchSize - padded
       for ((list, nd) <- outputs zip predExec.outputs) {
-        list += nd.slice(0, realSize).copy()
+        val ndSliced = nd.slice(0, realSize)
+        list += ndSliced.copy()
+        ndSliced.dispose()
```

Comment:
Shouldn't we do a try/finally to make sure dispose always happens, even in case of an exception?

Reply:
If we're getting an exception here something has gone very wrong. Most likely the problem would be memory access/allocation and I'm not confident dispose would work correctly under those conditions.

I'll make the change anyway because although I think it's unlikely to ever help it definitely won't hurt and is a good practice to follow.

Reply:
Feel a little bit overkilled adding try/finally for each time we dispose sth.
Eventually that should be handled by GC.